Team9Project/src/com/indragie/cmput301as1/ExpenseClaimListActivity.java

```diff
@@ -45,15 +43,23 @@
 	
 	private static final int ADD_EXPENSE_CLAIM_REQUEST = 1;
 	private static final int EDIT_EXPENSE_CLAIM_REQUEST = 2;
+	
 	private static final String EXPENSE_CLAIM_FILENAME = "claims";
 
 	//================================================================================
 	// Properties
 	//================================================================================
 	
-	private ArrayList<ExpenseClaim> claims;
-	private ExpenseClaimArrayAdapter adapter;
+	private static ArrayList<ExpenseClaim> claims;
```

Comment:
These shouldn't be static, otherwise it will share the same instances of these for every instance of `ExpenseClaimListActivity`